docs/source/quickstart.rst

```diff
@@ -17,7 +17,11 @@ Code
     optimizer = torch.optim.SGD(model.parameters(), lr=0.01, momentum=0.8))
     loss = torch.nn.NLLLoss()
 
-    trainer = create_supervised_trainer(model, optimizer, loss)
+    trainer = create_supervised_trainer(model, optimizer, criterion,
```

Comment:
Small typo here... `criterion` needs to be `loss` or vice versa

Reply:
thanks @jasonkriss !